contracts/test/accounts/LiquidityLockerOwner.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity 0.6.11;
+
+import { ILiquidityLocker } from "../../interfaces/ILiquidityLocker.sol";
+
+import { LiquidityLockerFactory } from "../../LiquidityLockerFactory.sol";
+
+contract LiquidityLockerOwner {
+
+    function liquidityLockerFactory_newLocker(address factory, address token) external returns (address) {
+        return address(ILiquidityLocker(LiquidityLockerFactory(factory).newLocker(token)));
```

Comment:
Can probably be `LiquidityLockerFactory(factory).newLocker(token)` - we should also use `ILiquidityLockerFactory`